packages/studio-server/src/routes.ts

```diff
@@ -195,6 +197,74 @@ const handleOpenInEditor = async (
 	}
 };
 
+const handleAddAsset = (
+	req: IncomingMessage,
+	res: ServerResponse,
+	publicDir: string,
+): void => {
+	const TWO_GIGABYTES = 2 * 1024 * 1024 * 1024;
+	const form = formidable({
+		maxTotalFileSize: TWO_GIGABYTES,
+		maxFieldsSize: TWO_GIGABYTES,
+		maxFileSize: TWO_GIGABYTES,
+	});
```

Comment:
Uploading a 10GB file does not seem to get rejected or display a useful error message